app/src/main/java/org/mozilla/fenix/tabhistory/TabHistoryViewHolder.kt

```diff
@@ -7,28 +7,34 @@ package org.mozilla.fenix.tabhistory
 import android.view.View
 import androidx.core.text.bold
 import androidx.core.text.buildSpannedString
-import androidx.core.view.isVisible
-import androidx.recyclerview.widget.RecyclerView
-import kotlinx.android.synthetic.main.history_list_item.view.*
+import kotlinx.android.synthetic.main.history_list_item.*
+import org.mozilla.fenix.library.LibrarySiteItemView
+import org.mozilla.fenix.utils.view.ViewHolder
 
 class TabHistoryViewHolder(
-    private val view: View,
+    view: View,
     private val interactor: TabHistoryViewInteractor
-) : RecyclerView.ViewHolder(view) {
+) : ViewHolder(view) {
+
+    private lateinit var item: TabHistoryItem
+
+    init {
+        itemView.setOnClickListener { interactor.goToHistoryItem(item) }
+    }
 
     fun bind(item: TabHistoryItem) {
-        view.history_layout.overflowView.isVisible = false
-        view.history_layout.urlView.text = item.url
-        view.history_layout.loadFavicon(item.url)
+        this.item = item
 
-        view.history_layout.titleView.text = if (item.isSelected) {
+        history_layout.displayAs(LibrarySiteItemView.ItemType.SITE)
```

Comment:
Oh cool I didn't know about this. But it shows the non-functional overflow menu. Maybe...
```suggestion
        history_layout.displayAs(LibrarySiteItemView.ItemType.SITE)
        history_layout.overflowView.isVisible = false
```

We might also want to update the padding for the title and url TextViews to account for the hidden overflow button though that could also be done in a future PR

Reply:
`displayAs` handles hiding the overflow button already, and the layout handles it

Reply:
I still see the overflow menu when I test this. Looking at the method it looks like the overflow menu is only hidden for separators

Reply:
Oops, sorry! I was mixing up components